<p align="center">
<img src="photo/IFOM.png"/>


<!-- PROJECT LOGO -->
<br />

  <h3 align="center"> <a href="https://www.ifom.eu/it/ricerca-cancro/ricerca-lab/ricerca-lab-scita.php"> <font size="+5">Scita Lab </font> </a> </h3>
  </a>
  <p align="center">
      <font size="+1">Mechanisms of Tumor Cell Migration Building 2, Floor 1, lab A </font>
  </p>
</p>

<!-- TABLE OF CONTENTS -->
<details open="open">
  <summary>Table of Contents</summary>
  <ol>
  <li>
      <a href="#about-the-lab">About the lab</a>
  </li>
  <li>
      <a href="#people">People</a>
  </li>
    <li>
      <a href="#pipelines">Pipelines</a>
    </li>
    <li>
      <a href="#scripts">Scripts</a>
    </li>
    <li>
      <a href="#publications"> Publications</a>
    </li>
    <li><a href="#contact">Contact</a></li>
  </ol>
</details>



<!-- ABOUT THE LAB -->
## About The Lab

The main focus of our team is 

* What are the molecular and physical mechanisms controlling tissue fluidification in epithelial cells and breast tumor models of DCIS?

* How the mechanical perturbations associated with unjamming by impacting the acquisition of heritable changes influence cell phenotypes, fate, and tumor ecology (tumor cell interaction with immune cells). We will specifically focus on unexpected cellular processes and molecular mechanisms underlying tissue fluidification-dependent genetic rewiring toward a cGAS-STING-mediated innate immunity response by studying the role of mitochondria dynamics, morphology, and integrity during unjamming-via-flocking.

* What are the pathobiological consequence of tissue fluidification by determining whether tissue fluidization can turn immunological cold lesions into hot ones using syngeneic mice models and human specimens.


<!-- people -->
## People

<p align="center">
<img src="photo/giorgio.jpeg" width="300" height="300" /> </p> 
<details><summary><a href="https://www.ifom.eu/en/cancer-research/researchers/giorgio-scita.php"> <font size="+1"> Giorgio Scita </font> </a> </summary>
</p align="justified"> 
A cell biologist and expert on the dynamics of cell movement, Giorgio Scita directs the Mechanisms of Tumor Cell Migration research unit at IFOM.
Born in 1963 near Parma, Scita enrolled in the Faculty of Biology at the University of Parma in 1982 with a precise intention: to study animal behavior alongside the famous Italian ethologist Danilo Mainardi.
</p>
</details>


<p align="center">
<img src="photo/AndreaP.jpg" width="300" height="300" />  </p>
<details><summary><a href="#Andrea_Palamidessi"><font size="+1"> Andrea Palamidessi </font> </a>
</summary>
</p align="justified"> 
Bio Andrea P.
</p>
</details>

<p align="center">
<img src="photo/Emy.jpg" width="300" height="300" />  </p>
<details><summary><a href="emanuela.frittoli@ifom.eu"><font size="+1"> Emanuela Frittoli </font> </a>
</summary>
</p align="justified"> 
My name is Emanuela Frittoli and I am a Cellular and Molecular Biologist. I earned my degree in Farmacist at “Università degli Studi di Milano” and my Techincal Biochemicl research diploma, at Mario Negri Institute. Then i'have moved Hospital San Raffaele in Paolo Lusso virology Lab. After i moved to the European Institute of Oncology (IEO) and at IFOM ETS, Milan, Italy, with of Prof. Giorgio Scita and Prof. Pier Paolo di Fiore.
</p>
</details>


<p align="center">
<img src="photo/AndreaD.jpg" width="300" height="300" />  </p>
<details><summary><a href="andrea.disanza@ifom.eu"><font size="+1"> Andrea Disanza </font> </a>
</summary>
</p align="justified"> 
My name is Andrea Disanza and I am a Cellular and Molecular Biologist. I earned my degree in Biological Sciences at “Università degli Studi di Milano” and my PhD in partnership with the Open University, London, UK, carrying out my research project at the European Institute of Oncology (IEO) and at IFOM ETS, Milan, Italy, under the supervision of Prof. Giorgio Scita and Prof. Pier Paolo di Fiore and at the LEBS, CNRS, Gif-sur-Yvette, France under the supervision of Dr. Marie France Carlier. I completed my scientific training with a two years Post Doc. Since 2008 I hold the position of Staff Scientist in the laboratory of Prof. Giorgio Scita “Mechanism of tumor cell migration” and the position of Teaching Assistant in General Patology at “Università degli Studi di Milano”, Dipartimento Scienze della Saute, Milan, Italy. My research experience ranges across several subjects: molecular biology, cell biology and biochemistry, biochemistry and actin biochemistry, ex vivo and in vivo mouse models and microscopy (Wide Field/Confocal/TIRF). My studies are focused on the interplay between membrane and actin dynamics in the control of migratory and invasive strategies in cancer with an insight in the role of the I-BAR containing protein IRSp53.
</p>
</details>


<p align="center">
<img src="photo/MonicaCorada.JPG" width="300" height="300" />  </p>
<details><summary><a href="monica.corada@ifom.eu"><font size="+1"> Monica Corada </font> </a>
</summary>
</p align="justified"> 
Bio Monica C.
</p>
</details>

<p align="center">
<img src="photo/Stefano.jpg" width="300" height="300" />  </p>
<details><summary><a href="stefano.marchesi@ifom.eu"><font size="+1"> Stefano Marchesi </font> </a>
</summary>
</p align="justified"> 
Stefano Marchesi got a degree in Molecular Cell Biology by the University of Milan in 2006. Then he was enrolled in the PhD Program of SEMM (European School of Molecular Medicine), working in the lab of Pier Paolo Di Fiore (IFOM-IEO Campus) on a project interconnecting cell cycle progression and adhesion signaling. After the completion of PhD course, he moved to the lab of Massimiliano Pagani at INGM (Istituto Nazionale di Genetica Molecolare). Successively he joined Politecnico di Milano as research grant, aiming to characterize the mechanobiology of glioblastoma. From 2019 on, he works as postdoc in the lab of Giorgio Scita at IFOM.
</p>
</details>

<p align="center">
<img src="photo/Hind.jpg" width="300" height="300" />  </p>
<details><summary><a href="hind.abdo@ifom.eu"><font size="+1"> Hind Abdo </font></a>
</summary>
</p align="justified"> 
Cell biologist by formation and a dreamer by nature, I grew up in Lebanon and ended up in Italy, after a long process of growth and maturation, first in France where I completed my PhD at the University of Nantes/Inserm Unit U913 then in Sweden at the Karolinska institute. Combining an extensive experience across several fields, I joined the laboratory of Prof. Giorgio Scita in 2020, converging my interest on molecular oncology. In particular, I am currently interested in studying the molecular and physical mechanisms controlling tissue fluidification in epithelial cells and the emergence of collective cell locomotion within solid tumors leading to the formation of metastasis.
</p>
</details>


<p align="center">
<img src="photo/Brenda.jpeg" width="300" height="300" />  </p>
<details><summary><a href="brenda.green@ifom.eu"><font size="+1"> Brenda Green </font> </a>
</summary>
</p align="justified"> 
 Brenda Green is a Biomedical Engineer and currently leading a project focused on developing micro-devices for capture of circulating tumor cells (CTCs) from whole blood, and for characterization of biophysical properties of tumor clusters within the lab of Giorgio Scita. She has worked closely with device design, fluidics and correlating tumor cellular assays with relevant biological insight, as a part of international research teams at IFOM, the University of Toronto and ETH Zurich. These teams have intersected with oncologists at Fondazione IRCCS Istituto Nazionale dei Tumori in Milan, and the Princess Margaret Hospital in Toronto to transition the technologies from laboratory to clinical research settings. At ETH Zurich, she demonstrated the advanced technological potential of a nano-printed system to track migration of cancer cells. Brenda obtained a bachelor degree in Chemical Engineering & Biochemistry from the University of Ottawa, a Master’s degree in Biomedical Engineering from the University of Toronto and a PhD in Biomedical Engineering from the University of Toronto. She completed her post-doc training in the lab of Giorgio Scita from 2019-2022 focused on design and development of a microfluidic system to capture circulating tumor cell clusters.
</p>
</details>

<p align="center">
<img src="photo/edobellini.jpg" width="300" height="300" />  </p>
<details><summary><a href="edoardo.bellini@ifom.eu"><font size="+1"> Edoardo Bellini </font> </a>
</summary>
</p align="justified"> 
Edoardo Bellini is a computational biologist and Bioinformatician working on basic research in the lab, and in the Bioinformatics Core of the <a href="https://www.ifom.eu/en/">IFOM </a> institute. His work is focused on all NGS data type coming from breast cancer disease models. He earned a Master Degree in Biology in 2018 at the University of Milano Bicocca and a Master in Bioinformatics and Functional Genomics in 2020 at University of Milan and INGM. He has gained previous research experiences as Bioinformatician at the San Raffaele Hospital analizyng all type of omics data and also as Computational Biologist as <a href="https://www.r-project.org/">R </a> package developer.
</p>
</details>


<p align="center">
<img src="photo/Ema.jpg" width="300" height="300" />
<details><summary><a href="emanuele.martini@ifom.eu"><font size="+1"> Emanuele Martini </font>  </a> </summary>
 </p> Bioimaging and biological data analysis specialist with 10 years of experience in developing analysis routines and workflows to extract meaningful biological knowledge from images and biological data, ML/AI included.  </p>
</details>


<p align="center">
<img src="photo/Leonardo.jpg" width="300" height="300" />
<details><summary><a href="leonardo.barzaghi@ifom.eu"><font size="+1">Leonardo Barzaghi </font> </a> </summary>
 </p> Leonardo Barzaghi earned his Bachelor's degree in Medical Biotechnology and later completed his Master's degree in Molecular Biotechnology and Bioinformatics at Università degli Studi di Milano. During his Master's program, he joined the research team led by Giorgio Scita, exploring the role of RAB5A protein family members in collective motility within epithelial cell monolayers. Intrigued by the multidisciplinary approaches used in his lab, Leonardo chose to continue his academic journey under the mentorship of Giorgio Scita, pursuing a Ph.D. in Molecular Oncology at SEMM. Now in his fourth year, Leonardo's research focus is at the intersection of cell biology and biophysics. He is specifically investigating the complex role of mechanical heterogeneity in breast cancer invasion, utilising 2D and 3D in vitro models. Leonardo's expertise includes live cell imaging and image analysis, with a particular emphasis on characterising collective cell motility at a biophysical level.  </p>
</details>

<p align="center">
<img src="photo/Camillo.jpg" width="300" height="300" />
<details><summary><a href="camillo.mazzella@ifom.eu"><font size="+1"> Camillo Mazzella </font>  </a> </summary>
 </p> Hi, I'Camillo Mazzella. I started as an engineer with a master’s degree in Industrial Bioengineering  Naples Federico II. 
Because I am interested in the oncology field, I completed a traineeship at ANGLE plc to model microfluidic devices for liquid biopsy. Then, I focused my career on oncology research to have better understand tumor biology. Actually, I'm a PhD student at IFOM, to study the impact of cell motility in cancer progression, within the group of Professor Giorgio Scita.  </p>
</details>



<p align="center">
<img src="photo/Mona.jpg" width="300" height="300" />
<details><summary><a href="mona.makkieh@ifom.eu"><font size="+1"> Mona Makkieh </font>  </a> </summary>
 </p> Mona Makkieh, research technician with a strong background in molecular oncology, actively contributing to cutting-edge research in Scita's group at IFOM, focusing on examining breast tumor cluster deformability as well as capturing circulating tumor cells. As an Erasmus Alumni, Mona spent a fruitful year as Student Researcher in the Department of Neuroscience at the University of Copenhagen, conducting structural studies on various G protein-coupled receptors (GPCRs) and investigating their correlation with cancer development. Regarding her educational background, Mona holds a Master's Degree in Molecular Biology and Genetics from the University of Pavia in Italy and a Bachelor's Degree in Medical Laboratory Technology from the Beirut Arab University in Lebanon. Mona's skill set is extensive and focused, covering DNA, RNA, protein, and plasmid purification, gene cloning, site-directed mutagenesis, microfluidics, spheroid assays, PCR/DNA sequencing, Southern/Western blotting, mammalian and insect cell cultures, cryo-sectioning, transfection, preparing membranes, saturation binding assay, competition binding assay, gel electrophoresis, and HPLC.  </p>
</details>


<p align="center">
<img src="photo/Davide.jpg" width="300" height="300" />
<details><summary><a href="davidelazzari7@gmail.com"> <font size="+1"> Davide Lazzari </font> </a> </summary>
 </p> Bio Davide </p>
</details>


<p align="center">
<img src="photo/Mattia.jpg" width="300" height="300" />
<details><summary><a href="davidelazzari7@gmail.com"> <font size="+1"> Mattia Tonani </font> </a> </summary>
 </p> Hi, i’m Mattia Tonani and I am student attending the medical biotechnology and molecular medicine master course at Universitá degli studi di Milano. Currently, I'm working on my 12 month internship thesis project at IFOM, under the supervision of Professor Giorgio Scita, aimed at discovering new parameters to be analyzed in a novel breast cancer’s mechano-classifier model based on machine learning, to predict the severity of the tumour. During this experience, I have been understanding the pure sense of biological and medical research, with an important focus on computational skills (Python and FIJI image analysis).  </p>
</details>


<!-- Pipelines -->
## Pipelines
In pipeline folder are present all our lab piplines in snakemake or nextflow. Within every folder are present:
* `.sk` or * `.nf` file, that contain all rules sequence. Usually for the standar pipelines the lab exploits the one in the <a href="https://nf-co.re/"> nf-core </a> repository. 
 ```bash 
 tail nextflow_rnaseq.sh

 nextflow run nf-core/rnaseq \
              --outdir output \
	            -profile slurm,singularity \
	            -c /path/to/ifom.config \
	            --input /path/to/my_samplesheet_test.csv \
	            --fasta /path/to/Homo_sapiens.GRCh38.dna.primary_assembly.fa \
              --gtf /path/to/Homo_sapiens.GRCh38.104.gtf \
	            -r 3.13.2 \
              --save_reference #\
```
* `.csv` file conteins all information abount the metadata of the samples such as sample condition, or whatever you need to specify.
```bash
cat samplesheet.csv

sample,fastq_1,fastq_2,strandedness
CTRL_1,/path/to/fastq/NG-33919_1_lib709956_10277_4_1.fastq.gz,/path/to/fastq/NG-33919_1_lib709956_10277_4_2.fastq.gz,auto
FLOW_ONLY_1,/path/to/fastq/NG-33919_2_lib709957_10277_4_1.fastq.gz,/path/to/fastq/NG-33919_2_lib709957_10277_4_2.fastq.gz,auto
DEFORMED_1,/path/to/fastq/NG-33919_3_lib709958_10277_4_1.fastq.gz,/path/to/fastq/NG-33919_3_lib709958_10277_4_2.fastq.gz,auto
CTRL_2,/path/to/fastq/NG-33919_4_lib709959_10277_4_1.fastq.gz,/path/to/fastq/NG-33919_4_lib709959_10277_4_2.fastq.gz,auto
FLOW_ONLY_2,/path/to/fastq/NG-33919_5_lib709960_10277_4_1.fastq.gz,/path/to/fastq/NG-33919_5_lib709960_10277_4_2.fastq.gz,auto
DEFORMED_2,/path/to/fastq/NG-33919_6_lib709961_10277_4_1.fastq.gz,/path/to/fastq/NG-33919_6_lib709961_10277_4_2.fastq.gz,auto
CTRL_3,/path/to/fastq/NG-33919_7_lib709962_10277_4_1.fastq.gz,/path/to/fastq/NG-33919_7_lib709962_10277_4_2.fastq.gz,auto
FLOW_ONLY_3,/path/to/fastq/NG-33919_8_lib709963_10277_4_1.fastq.gz,/path/to/fastq/NG-33919_8_lib709963_10277_4_2.fastq.gz,auto
DEFORMED_3,/path/to/fastq/NG-33919_9_lib709964_10277_4_1.fastq.gz,/path/to/fastq/NG-33919_9_lib709964_10277_4_2.fastq.gz,auto
```

<!-- SCRIPTS -->
## Scripts
In scripts folder are present stand alone script used generally to perform downstream analisys like scRNA-seq clustering, Bulk RNA-seq, or custom plot.

<!-- PUBLICATIONS -->
## Publications

<ol>
  <li>
      <div class="references">
      <p> <b> Frittoli E  </b>, <b> Palamidessi A </b> </a>, Iannelli F, Zanardi F, Villa S, Barzaghi L,  <b> Abdo H  </b> , Cancila V, Beznoussenko GV, Della Chiara G, Pagani M, Malinverno C, Bhattacharya D, Pisati F, Yu W, Galimberti V, Bonizzi G, Martini E, Mironov AA, Gioia U, Ascione F, Li Q, Havas K, Magni S, Lavagnino Z, Pennacchio FA, Maiuri P, Caponi S, Mattarelli M, Martino S, d'Adda di Fagagna F, Rossi C, Lucioni M, Tancredi R, Pedrazzoli P, Vecchione A, Petrini C, Ferrari F, Lanzuolo C, Bertalot G, Nader G, Foiani M, Piel M, Cerbino R, Giavazzi F, Tripodo C,  <b>Scita G </b> .
      <b>"Tissue fluidification promotes a cGAS-STING cytosolic DNA response in invasive breast cancer"</b>.
      <em>Nat Mater. 2022 Dec 29. doi: 10.1038/s41563-022-01431-x. Online ahead of print.PMID: 36581770 </em>. <p align="center">  <a href="https://www.nature.com/articles/s41563-022-01431-x"> <img src="photo/1.png" width="100" height="100" > </a> </p>
      </p> 
      </div>
  </li>

  <li>
      <div class="references">
      <p>Nader GPF, Agüera-Gonzalez S, Routet F, Gratia M, Maurin M, Cancila V, Cadart C, <b> Palamidessi  A </b>, Ramos RN, San Roman M, Gentili M, Yamada A, Williart A, Lodillinsky C, Lagoutte E, Villard C, Viovy JL, Tripodo C, Galon J, <b> Scita G </b>, Manel N, Chavrier P, Piel M. <b>"Compromised nuclear envelope integrity drives TREX1-dependent DNA damage and tumor cell invasion"</b>.<em>Cell. 2021 Sep 30;184(20):5230-5246.e22. doi: 10.1016/j.cell.2021.08.035. Epub 2021 Sep 21. PMID: 34551315. </em>. <p align="center"> <a href="https://www.cell.com/cell/fulltext/S0092-8674(21)01046-1"> <img src="photo/2.png" width="100" height="100" > </a> </p> 
      </p>
    </div>
  </li>
  
  <li>
      <div class="references">
      <p> <b> Palamidessi A </b>, Malinverno C, <b>Frittoli E </b>, Corallino S, Barbieri E, Sigismund S, Beznoussenko GV, <b> Martini E </b>, Garre M, Ferrara I, Tripodo C, Ascione F, Cavalcanti-Adam EA, Li Q, Di Fiore PP, Parazzoli D, Giavazzi F, Cerbino R, <b> Scita G </b>. <b> "Unjamming overcomes kinetic and proliferation arrest in terminally differentiated cells and promotes collective motility of carcinoma"</b>. <em> Nat Mater. 2019 Nov;18(11):1252-1263. doi: 10.1038/s41563-019-0425-1. Epub 2019 Jul 22. Erratum in: Nat Mater. 2022 Dec;21(12):1448. PMID: 31332337.</em> <p align="center"> <a href="https://www.nature.com/articles/s41563-019-0425-1"> <img src="photo/3.png" width="100" height="100"> </a> </p>
      </p>
      </div>
  </li>

  <li>
      <div class="references">
      <p> Bisi S, <b> Marchesi S </b>, Rizvi A, Carra D, Beznoussenko GV, Ferrara I, Deflorian G, Mironov A, Bertalot G, Pisati F, Oldani A, Cattaneo A, Saberamoli G, Pece S, Viale G, Bachi A, Tripodo C, <b> Scita G </b>, <b> Disanza A </b>. <b>"IRSp53 controls plasma membrane shape and polarized transport at the nascent lumen in epithelial tubules".</b> <em> Nat Commun. 2020 Jul 14;11(1):3516. doi: 10.1038/s41467-020-17091-x. PMID: 32665580; PMCID: PMC7360740.</em> <p align="center"> <a href="https://www.nature.com/articles/s41467-020-17091-x"> <img src="photo/4.png" width="100" height="100"> </a> </p>
      </p>
      </div>
  </li>

  <li>
      <div class="references">
      <p> Corallino S, Malinverno C, Neumann B, Tischer C, <b> Palamidessi A, Frittoli E, </b> Panagiotakopoulou M, <b> Disanza A </b>, Malet-Engra G, Nastaly P, Galli C, Luise C, Bertalot G, Pece S, Di Fiore PP, Gauthier N, Ferrari A, Maiuri P, <b> Scita G </b>. <b> "A RAB35-p85/PI3K axis controls oscillatory apical protrusions required for efficient chemotactic migration" </b>. <em> Nat Commun. 2018 Apr 16;9(1):1475. doi: 10.1038/s41467-018-03571-8. Erratum in: Nat Commun. 2018 May 22;9(1):2085. PMID: 29662076; PMCID: PMC5902610.</em> <p align="center"> <a href="https://www.nature.com/articles/s41467-018-03571-8"> <img src="photo/5.png" width="100" height="100"> </a> </p>
      </p>
      </div>
  </li>

  <li>
      <div class="references">
      <p> Zobel M, <b> Disanza A </b>, Senic-Matuglia F, Franco M, Colaluca IN, Confalonieri S, Bisi S, Barbieri E, Caldieri G, Sigismund S, Pece S, Chavrier P, Di Fiore PP, <b> Scita G </b>. <b> "A NUMB-EFA6B-ARF6 recycling route controls apically restricted cell protrusions and mesenchymal motility" </b>. <em> J Cell Biol. 2018 Sep 3;217(9):3161-3182. doi: 10.1083/jcb.201802023. Epub 2018 Jul 30. PMID: 30061108; PMCID: PMC6123001.</em> <p align="center"> <a href="https://rupress.org/jcb/article/217/9/3161/120841/A-NUMB-EFA6B-ARF6-recycling-route-controls"> <img src="photo/6.png" width="100" height="100"> </a> </p>
      </p>
      </div>
  </li>

  <li>
      <div class="references">
      <p> Malinverno C, Corallino S, Giavazzi F, Bergert M, Li Q, Leoni M,  <b> Disanza A, Frittoli E, </b> Oldani A,  <b> Martini E  </b>, Lendenmann T, Deflorian G, Beznoussenko GV, Poulikakos D, Haur OK, Uroz M, Trepat X, Parazzoli D, Maiuri P, Yu W, Ferrari A, Cerbino R,  <b> Scita G  </b>.  <b> "Endocytic reawakening of motility in jammed epithelia"  </b>. <em> Nat Mater. 2017 May;16(5):587-596. doi: 10.1038/nmat4848. Epub 2017 Jan 30. PMID: 28135264; PMCID: PMC5407454.</em> <p align="center">  <a href="https://www.nature.com/articles/nmat4848"> <img src="photo/7.png" width="100" height="100" > </a> </p>
      </p>
      </div>
  </li>

  <li>
      <div class="references">
      <p> Kajiho H, Kajiho Y,<b> Frittoli E </b>, Confalonieri S, Bertalot G, Viale G, Di Fiore PP, Oldani A, Garre M, Beznoussenko GV, <b> Palamidessi A </b>, Vecchi M, Chavrier P, Perez F, <b>Scita G</b>. "RAB2A controls MT1-MMP endocytic and E-cadherin polarized Golgi trafficking to promote invasive breast cancer programs"</b>. <em> EMBO Rep. 2016 Jul;17(7):1061-80. doi: 10.15252/embr.201642032. Epub 2016 Jun 2. PMID: 27255086; PMCID: PMC4931572.</em> <p align="center"> <a href="https://www.embopress.org/doi/full/10.15252/embr.201642032"> <img src="photo/8.png" width="100" height="100" > </a> </p>
      </p>
      </div>
  </li>

  <li>
      <div class="references">
      <p> Giampietro C, <b> Disanza A </b>, Bravi L, Barrios-Rodiles M, Corada M, <b> Frittoli E </b>, Savorani C, Lampugnani MG, Boggetti B, Niessen C, Wrana JL, <b> Scita G </b>, Dejana E. <b> "The actin-binding protein EPS8 binds VE-cadherin and modulates YAP localization and signaling" </b>. <em> J Cell Biol. 2015 Dec 21;211(6):1177-92. doi: 10.1083/jcb.201501089. Epub 2015 Dec 14. PMID: 26668327; PMCID: PMC4687874.</em> <p align="center"> <a href="https://rupress.org/jcb/article/211/6/1177/38251/The-actin-binding-protein-EPS8-binds-VE-cadherin"> <img src="photo/9.png" width="100" height="100" > </a> </p>
      </p>
      </div>
  </li>

  <li>
      <div class="references">
      <p> Rolland Y, Marighetti P, Malinverno C, Confalonieri S, Luise C, Ducano N, <b> Palamidessi A </b>, Bisi S, Kajiho H, Troglio F, Shcherbakova OG, Dunn AR, Oldani A, Lanzetti L, Di Fiore PP,<b> Disanza A, Scita G </b>. <b> "The CDC42-interacting protein 4 controls epithelial cell cohesion and tumor dissemination" </b>. <em> Dev Cell. 2014 Sep 8;30(5):553-68. doi: 10.1016/j.devcel.2014.08.006. PMID: 25203208.</em> <p align="center"> <a href="https://www.cell.com/developmental-cell/fulltext/S1534-5807(14)00516-4"> <img src="photo/10.png" width="100" height="100"> </a> </p>
      </p>
      </div>
  </li>

  <li>
      <div class="references">
      <p> <b> Frittoli E., A. Palamidessi </b>, P. Marighetti, S. Confalonieri, F. Bianchi, C. Malinverno, G. Mazzarol, G. Viale, I. Martin-Padura, M. Garre, D. Parazzoli, V. Mattei, S. Cortellino, G. Bertalot, P.P. Di Fiore, and <b> G. Scita. 2014 </b>. <b> "A RAB5/RAB4 recycling circuitry induces a proteolytic invasive program and promotes tumor dissemination" </b>. <em> J Cell Biol. 206:307-328.</em> <p align="center"> <a href="https://rupress.org/jcb/article/206/2/307/37826/A-RAB5-RAB4-recycling-circuitry-induces-a"> <img src="photo/11.png" width="100" height="100" > </a> </p>
      </p>
      </div>
  </li>
</ol>  


<!-- CONTACT -->
## Contacts


Edoardo Bellini - <edoardo.bellini@ifom.eu>

Giorgio Scita - <giorgio.scita@ifom.eu>

<!-- MARKDOWN LINKS & IMAGES -->
<!-- https://www.markdownguide.org/basic-syntax/#reference-style-links -->
[contributors-shield]: https://img.shields.io/github/contributors/othneildrew/Best-README-Template.svg?style=for-the-badge
[contributors-url]: https://github.com/othneildrew/Best-README-Template/graphs/contributors
[forks-shield]: https://img.shields.io/github/forks/othneildrew/Best-README-Template.svg?style=for-the-badge
[forks-url]: https://github.com/othneildrew/Best-README-Template/network/members
[stars-shield]: https://img.shields.io/github/stars/othneildrew/Best-README-Template.svg?style=for-the-badge
[stars-url]: https://github.com/othneildrew/Best-README-Template/stargazers
[issues-shield]: https://img.shields.io/github/issues/othneildrew/Best-README-Template.svg?style=for-the-badge
[issues-url]: https://github.com/othneildrew/Best-README-Template/issues
[license-shield]: https://img.shields.io/github/license/othneildrew/Best-README-Template.svg?style=for-the-badge
[license-url]: https://github.com/othneildrew/Best-README-Template/blob/master/LICENSE.txt
[linkedin-shield]: https://img.shields.io/badge/-LinkedIn-black.svg?style=for-the-badge&logo=linkedin&colorB=555
[linkedin-url]: https://linkedin.com/in/othneildrew
[product-screenshot]: images/screenshot.png
